src/main/java/org/zalando/nakadi/domain/EventTypeOptions.java

```diff
@@ -4,13 +4,13 @@
 
 public class EventTypeOptions {
 
-    @Nullable
     private Long retentionTime;
 
     public Long getRetentionTime() {
         return retentionTime;
     }
 
+    @Nullable
```

Comment:
I think you put it to the wrong place. You should have Nullable on the getter and on parameter of setter.